config/application.rb

```diff
@@ -16,6 +16,24 @@
 
 module FakeNewsApi
   class Application < Rails::Application
+    config.generators do |generate|
+      generate.helper false
+      generate.assets false
+      generate.view_specs false
+      generate.helper_specs false
+      generate.routing_specs false
+      generate.controller_specs false
+    end
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
+        resource '*', 
+          headers: :any, 
+          methods: %i[get post put delete],
+          expose: %w(access-token expiry token-type uid client),
+          max_age: 0
+      end
```

Comment:
Why are you adding configuration for the rack-cors in this PR?

Reply:
We had a change of heart about this. We read up on the documentation and have decided to add it later when it's needed.